NFC rename consumed_mah

amilcarlucas opened the pull request:
This is a followup to #7441

dgrat commented:
I think it sounds silly. You "consume" butter. If you want to rename, then at least to "integrated_mAh"

amilcarlucas commented:
It is better than the current name
It makes it consistent with `consumed_wh`

dgrat commented:
Okay, I agree that current_total_* is not good. Why not total_*?

amilcarlucas commented:
* total consumed ?
* total available ?

Which one is it ? That is the reason that consumed was chosen instead of total_consumed.

amilcarlucas commented:
I do not understand your comment. Can you please point to a specific line of code ?

amilcarlucas commented on libraries/AP_BattMonitor/AP_BattMonitor.h:
```diff
@@ -52,14 +52,14 @@ class AP_BattMonitor
-        float       current_amps;       // current in amperes
-        float       current_total_mah;  // total current draw since start-up
-        float       consumed_wh;        // total energy consumed in Wh since start-up
+        float       voltage;            // voltage in Volt
```
@WickedShell are you talking about this hunk ?

amilcarlucas replied:
@WickedShell are you talking about this specific hunk ??

amilcarlucas commented:
I corrected it to consistently use SI standard names and conventions

olliw42 commented:
mAh is NOT a SI standard names and conventions !!!
and Wh is neither
and a . doesn't make it different
:)

EShamaev commented:
Just out of curiosity, what symbol would you propose instead of half-high dot?

WickedShell commented on libraries/GCS_MAVLink/GCS_Common.cpp:
```diff
@@ -199,9 +199,9 @@ void GCS_MAVLINK::send_battery_status(const AP_BattMonitor &battery, const uint8
-                                    battery.has_current(instance) ? battery.current_amps(instance) * 100 : -1, // current
-                                    battery.has_current(instance) ? battery.current_total_mah(instance) : -1, // total current
+                                    battery.get_cell_voltages(instance).cells, // cell voltages in mV
+                                    battery.has_current(instance) ? battery.current_amps(instance) * 100 : -1, // current in cA
```
Stuff like this is an absolutely awful way to note units, IMO. This is what I'm referring to, it should either be written out fully (or in this case can probably be ignored since this is easy to read). In general I think adding the .'s/capitalization in a bunch of places has done nothing to improve the readability and in a lot of ways made it worse.

amilcarlucas commented:
@olliw42 I do not agree with your affirmations. My changes to the code come from the following documents:
# http://physics.nist.gov/cuu/Units/units.html
# http://physics.nist.gov/cuu/Units/outside.html
# and
# http://physics.nist.gov/cuu/Units/checklist.html
# http://www.bipm.org/en/publications/si-brochure/
# http://www1.bipm.org/en/CGPM/db/3/2/   g_n unit for G-force

What information sources are you using to make those claims ?

You do understand that the goal of the Ardupilot Project is not to restrict the use to main SI units only ?
Derived SI units ar also valid i.e. Km/h is valid although m/s is the correct one.
Misspelled units are NOT valid, and THAT is what I and other project members do not like.
Derived units are OK.

Is that clear for you ?

@WickedShell I addressed all your points now.

amilcarlucas commented:
@OXINARF I addressed your comments.

OXINARF commented:
@amilcarlucas It was much better, thanks! I didn't merge the change of division to multiplication, I already find that almost useless, I'm totally against making a commit just for it.

Reworded commit messages and merged, thanks!
